src/styles/components/pricing-card.scss

```diff
@@ -0,0 +1,52 @@
+.pricing-card {
```

Comment:
As far as I understand you've decided to keep all styles in separate css files?

Reply:
Yes, I create separate files for components or pages